pkg/controller/factory/builder.go

```diff
@@ -87,6 +87,7 @@ func BuildInstanceSet(synthesizedComp *component.SynthesizedComponent, component
 		AddLabelsInMap(compDefLabel).
 		AddLabelsInMap(constant.GetAppVersionLabel(compDefName)).
 		AddLabelsInMap(synthesizedComp.UserDefinedLabels).
+		AddLabelsInMap(constant.GetClusterWellKnownLabels(clusterName)).
```

Comment:
is there any other label needed to show correctly on grafana?

Reply:
Not, the dashboard only needs these labels: namespace, pod and cluster.